$NetBSD: patch-configure,v 1.1 2021/03/19 21:48:27 markd Exp $

* use std::regex instead of boost that don't have regex since 1.65
  from FreeBSD ports 

--- configure.orig	2021-03-18 09:51:20.788551616 +0000
+++ configure
@@ -17554,13 +17554,13 @@ cat confdefs.h - <<_ACEOF >conftest.$ac_
 /* end confdefs.h.  */
 
 
-#include <boost/tr1/regex.hpp>
+#include <regex>
 
 int
 main ()
 {
-  std::tr1::regex r ("te.t", std::tr1::regex_constants::ECMAScript);
-  return std::tr1::regex_match ("test", r) ? 0 : 1;
+  std::regex r ("te.t", std::regex_constants::ECMAScript);
+  return std::regex_match ("test", r) ? 0 : 1;
 }
 
 _ACEOF
@@ -17631,13 +17631,13 @@ cat confdefs.h - <<_ACEOF >conftest.$ac_
 /* end confdefs.h.  */
 
 
-#include <boost/tr1/regex.hpp>
+#include <regex>
 
 int
 main ()
 {
-  std::tr1::regex r ("te.t", std::tr1::regex_constants::ECMAScript);
-  return std::tr1::regex_match ("test", r) ? 0 : 1;
+  std::regex r ("te.t", std::regex_constants::ECMAScript);
+  return std::regex_match ("test", r) ? 0 : 1;
 }
 
 _ACEOF
